packages/oruga-next/src/components/sidebar/Sidebar.vue

```diff
@@ -334,15 +334,18 @@ onBeforeUnmount(() => {
     }
 });
 
+let removeOutsideListener = null;
+
 /** add outside click event listener */
 function addHandler(): void {
-    if (isClient && !props.overlay)
-        setTimeout(() => document.addEventListener("click", clickedOutside));
+    if (isClient && !props.overlay) {
+        removeOutsideListener = useClickOutside(rootRef, clickedOutside);
```

Comment:
What does this change have to do with the Bulma integration? Did you fix some other bug here? maybe exclude it in a second PR :)

Reply:
I mentioned this in the PR description. There was a unrelated bug where the sidebar closed itself whenever you clicked on just about anything, including the contents of the sidebar. I should be able to break this out.

Reply:
Now PR https://github.com/oruga-ui/oruga/pull/786